pytorch_lightning/profiler/simple.py

```diff
@@ -72,13 +72,21 @@ def stop(self, action_name: str) -> None:
         duration = end_time - start_time
         self.recorded_durations[action_name].append(duration)
 
-    def _make_report(self) -> Tuple[list, float]:
+    def _make_report_extended(self) -> Tuple[List, float, float]:
```

Comment:
Annotate the list types? You can use variables to make it easier to read.

Reply:
oh yes.. haha. was going to but forgot to update this...

Reply:
btw, what do you mean by variables here?

Reply:
Something like:

```python
_TABLE_ROW = Tuple[...]
_TABLE_DATA = List[_TABLE_ROW] 

def _make_report_extended(self) -> Tuple[_TABLE_DATA, float, float]:
```

Reply:
done!

Reply:
Beautiful!